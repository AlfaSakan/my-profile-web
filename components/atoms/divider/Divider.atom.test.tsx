import renderer from 'react-test-renderer';
import Divider from './Divider.atom';

describe('divider component test', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Divider />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
