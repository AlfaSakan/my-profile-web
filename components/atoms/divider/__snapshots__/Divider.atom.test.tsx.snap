// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divider component test should render correctly 1`] = `
<div
  className="border-[0.5px] w-[calc(100%-87px)] self-end mr-3"
/>
`;
